tests/Lykke.Service.FeeCalculator.Tests/Modules/TestsModule.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System.Linq;
+using Autofac;
+using AzureStorage.Tables;
+using Common;
+using Common.Log;
+using Lykke.Service.ClientAccount.Client;
+using Lykke.Service.FeeCalculator.AzureRepositories.Fees;
+using Lykke.Service.FeeCalculator.Core.Domain.Fees;
+using Lykke.Service.FeeCalculator.Core.Services;
+using Lykke.Service.FeeCalculator.Services;
+using Lykke.Service.TradeVolumes.Client;
+using Moq;
+
+namespace Lykke.Service.FeeCalculator.Tests.Modules
+{
+    public class TestsModule : Module
+    {
+        protected override void Load(ContainerBuilder builder)
+        {
+            builder.RegisterInstance(Mock.Of<ILog>())
+                .As<ILog>()
+                .SingleInstance();
+
+            builder.RegisterType<TradeVolumesCacheService>()
+                .As<ITradeVolumesCacheService>()
+                .SingleInstance();
+            
+            var mockTradeVolumesClient = new Mock<ITradeVolumesClient>();
+            
+            builder.RegisterInstance(mockTradeVolumesClient)
+                .SingleInstance();
+            
+            builder.RegisterInstance(mockTradeVolumesClient.Object)
+                .SingleInstance();
+            
+            builder.RegisterInstance(
+                GetFeeRepository()  
+            ).As<IFeeRepository>().SingleInstance();
+            
+            builder.RegisterInstance(
+                GetStaticFeeRepository()  
+            ).As<IStaticFeeRepository>().SingleInstance();
+            
+            builder.Register(x =>
+            {
+                var ctx = x.Resolve<IComponentContext>();
+                return new CachedDataDictionary<decimal, IFee>(
+                    async () => (await ctx.Resolve<IFeeRepository>().GetFeesAsync()).ToDictionary(itm => itm.Volume));
+            }).SingleInstance();
+            
+            builder.Register(x =>
+            {
+                var ctx = x.Resolve<IComponentContext>();
+                return new CachedDataDictionary<string, IStaticFee>(
+                    async () => (await ctx.Resolve<IStaticFeeRepository>().GetFeesAsync()).ToDictionary(itm => itm.AssetPair));
+            }).SingleInstance();
+            
+            builder.RegisterType<FeeService>()
+                .As<IFeeService>()
+                .SingleInstance();
+            
+            builder.RegisterInstance(Mock.Of<IClientAccountClient>())
+                .As<IClientAccountClient>()
+                .SingleInstance();
+            
+            builder.RegisterType<ClientIdCacheService>()
+                .As<IClientIdCacheService>()
+                .SingleInstance();
+        }
+        
+        private IFeeRepository GetFeeRepository()
+        {
+            var repository = new FeeRepository(new NoSqlTableInMemory<FeeEntity>());
+            
+            repository.AddFeeAsync(new Fee{Volume = 0.1M, MakerFee = 0.1M, TakerFee = 0.19M}).Wait();
```

Comment:
Use GetAwaiter().GetResult()
Probably it is better to add a method with 3 arguments to reduce copy&paste